### PR TITLE
API Support tika server

### PR DESCRIPTION
**Pull request by tractorcow**

### CODE REVIEW

**Comment by tractorcow** [comment]
This needs some attention, @chillu @halkyon :D

File: README.md [hunk 1]
```diff
@@ -132,15 +132,54 @@ configuring this.
...
+```bash
+#!/bin/bash
+exec java -jar /usr/local/Cellar/tika/1.7/libexec/tika-app-1.7.jar "$@"
+```
+
+### Tika Rest Server
+
+Tika can also be run as a server.
+
+You can configure your server pointpoint either by the SS_TIKA_ENDPOINT define, or by setting the url via config.
```
Comment (chillu) [inline README.md]: Spelling mistake (will fix in follow up commit)

File: code/extractors/TikaServerTextExtractor.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+				)
+			);
+	}
+
+	public function getServerEndpoint() {
+		if(defined('SS_TIKA_ENDPOINT')) {
+			return SS_TIKA_ENDPOINT;
+		}
+
+		if(getenv('SS_TIKA_ENDPOINT')) return getenv('SS_TIKA_ENDPOINT');
```
Comment (chillu) [inline code/extractors/TikaServerTextExtractor.php]: Bracketing is a bit inconsistent here

**Comment by chillu** [comment]
Legend!

**Pull request merged**